src/middlewares/cors.ts

```diff
@@ -0,0 +1,7 @@
+import { Request, Response, NextFunction } from 'express';
+
+export function setCORSHeaders(request: Request, response: Response, next: NextFunction) {
+  response.header('Access-Control-Allow-Origin', '*');
+  response.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+  next();
+}
```

Comment:
there's a package called [`cors`](https://www.npmjs.com/package/cors) that does exactly that. Any reason not to use that?